app/controllers/organizations/staff/dashboard_controller.rb

```diff
@@ -22,30 +23,21 @@ def incomplete_tasks
       items: 5
     )
     @column_name = "Incomplete Tasks"
+    @header_title = "Incomplete Table"
```

Comment:
Could label this `Incomplete Pet Tasks`?

Reply:
Done